src/components/pages/ParentBooking/ParentBookingContainer.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useState } from 'react';
+import 'antd/dist/antd.css';
+import { Card } from 'antd';
+import { connect } from 'react-redux';
+import { Tabs } from 'antd';
+import ParentBookingCard from './ParentBookingCard';
+import LoadingComponent from '../../common/LoadingComponent';
+
+function isDateInThisWeek(date) {
+  const todayObj = new Date();
+  const todayDate = todayObj.getDate();
+  const todayDay = todayObj.getDay();
+
+  const firstDayOfWeek = new Date(todayObj.setDate(todayDate - todayDay));
+  const lastDayOfWeek = new Date(firstDayOfWeek);
+  lastDayOfWeek.setDate(lastDayOfWeek.getDate() + 6);
+
+  return firstDayOfWeek <= new Date(date) && new Date(date) <= lastDayOfWeek;
+}
+
+const tabs = [{ title: 'All' }, { title: 'Today' }, { title: 'This Week' }];
+
+const ParenBookingContainer = props => {
+  const { TabPane } = Tabs;
+  const { bookings, isFetching, error } = props.bookings;
+  const [currentTab, SetcurrentTab] = useState(null);
+
+  useEffect(() => {
+    // when the component mounts, trigger action fetchParentBookings to get the list of bookings
+    // props.fetchParentBookings();
+    SetcurrentTab(bookings);
+  }, []);
+
+  const renderTab = key => {
```

Comment:
Excellent work, very clean, and I love this function more than anything. It's brilliant.